### DISCUSSION TITLE
求资源丰富的动漫网址

### DISCUSSION THREAD

**Discussion by AltoriaPandoragon** [share121/inter-knot#2794] [其他]
好心人！给小的推荐的网站吧！

**Comment by cpu644** [comment]
bilibili

**Comment by shan-dgha** [comment]
https://www.mengdaow.com/<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Comment by chenx198** [comment]
https://www.olevod.com/index.php/vod/show/id/4.html
